docs/user/kubevirt/install-config.yaml

```diff
@@ -0,0 +1,37 @@
+sshKey: |
+  ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQD4vtFWqLK9sNGVrTUXQEeRY5iFz/MF3IuFzWmEOtF1xG/m5qEPctwWNhH7g90IMnsdHjdvcAmEEcaeFlVrABq6t529Xwy7eqXRD1B2Xx5rhr2TisnWqPC8gEzIrp1cIvcCG1fxLdNRyNU1+aylDi6vnCezbp7GwZnqcy0krxpwsJ+5gaz7euUXRzvG+KXdc2p5wiJKn5Ld88cD4OBzGEr3n0MagbZhl5o5tgrFRJdlaEHZgOD927Fsd4edKTWkacYwpcZvYtCEIKeHa97ITN8Jxom73snfGWdz3Mg4nGpRBH2Elz0lnGWN4IfVjw9PLQiQ5949n1qoJq91TX3DutKBJ5tnrCoOBjT4lB7FttvFOINsR+bEszb2YVpX+UMOAayXvs7Nz1pdFzgUQzY3TR5W9EJ4kLm3BK3+piCHL44hgEQoYMMOM4dvx/vnzCb5WlGEMvOi//j60HQqmxhv2kH7FW4SwyNTMVAZ9fSnTSZKWKj5OWxGQXNhw+K2RNx+8RZ3xTN29+qqR5Mk2titRKFBvyg+Nox9G2PCpzQ7DA5ZGhrEJr/GmxWt/HvrrVVCLIQ1tYivKjMtjL8UJ9kEtccWaBfml6BgjOWQBAcW2+m90MxcFXtIA36xqbLJFDxPFG22X0LsZnm4DMVU2cUL5lDBUgTDyoUiTVEi0RGo9iif+w== nargaman@redhat.com
```

Comment:
I don't think you want you personal pull secret and ssh key here .. (-:

Reply:
OOPS :-)
Fixed ... thanks